### ISSUE TITLE
Cursor keys don't work in annotation editor list

### ISSUE THREAD

**Issue by zack4200** [sumatrapdfreader/sumatrapdf#3770]
similar to issue https://github.com/sumatrapdfreader/sumatrapdf/issues/3685, except for using the arrow keys to move between different annotations in the list. They work in 3.4.6 but each of the pre-release versions I've used the arrows *only* work in the Contents area.

**Comment by kjk** [comment]
Yeah, people wanted auto-focus in content text field, which is why cursor keys do not navigate the list.

Maybe I can support this in a hacky way or un-do the auto-focus on content field.

**Comment by GitHubRulesOK** [comment]
the up down works in list if you click on list.

**Comment by GitHubRulesOK** [comment]
@kjk  since most recent change has been to improve focus into content this request is related to select an annot and focus goes to their content but perhaps a non printing hotkey such as CTRL + Tab (NOT AltTab) that does not affect entering a tab space could be used to swap focus. Personally I just click the upper box and arrows work

**Comment by GitHubRulesOK** [comment]
@kjk seems resolved on open editor it goes to highlight and up down keys stay in the top box

<img width="812" height="231" alt="Image" src="https://github.com/user-attachments/assets/d6c1ab9f-625d-4fdb-8188-7943e3bad360" />

**Comment by kjk** [comment]
yes, seems to work in latest pre-release
